sdks/node/src/grpc/chunks.ts

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-ts_proto. DO NOT EDIT.
 // versions:
-//   protoc-gen-ts_proto  v2.0.2
-//   protoc               v3.21.12
+//   protoc-gen-ts_proto  v2.0.3
```

Comment:
Can you regenerate in `devcontainer`?
The versions are not sync with `devcontainer` package version